model/schema/gene.yaml

```diff
@@ -102,8 +102,8 @@ classes:
       single_gene:
         required: true
       name_type:
-        any_of:
-          - equals_string: nomenclature_synonym
+        name_type:
+          notes: value must be VocabularyTerm with name "symbol"
```

Comment:
@gildossantos - Is this correct?  The `any_of` rule previously stated it should be `nomenclature_synonym`, but the  `AlleleSymbolSlotAnnotation` states it should be `symbol`

Reply:
@markquintontulloch - sorry, that was a typo. It should be "nomenclature_symbol". I've updated the `SymbolSlotAnnotationDTO`, `AlleleSymbolSlotAnnotation` and `GeneSymbolSlotAnnotation` objects accordingly.